functions/helpFunctions.tsx

```diff
@@ -316,3 +317,9 @@ function excludeArray2fromArray1(array1: AnswerType[], array2: AssessmentType[])
     );
   });
 }
+
+// check if all scores are set
+export function checkScores(assessments: AssessmentType[]): boolean {
+  return !assessments.some((e) => e.score === '');
+  return status;
```

Comment:
You forgot the `return status` ! I didn't include it in the suggestion since I had selected the wrong part in the first comment 🙈

Reply:
#123